recipes/grapharray/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The LICENSE is included in the sdist on pypi. No need to put it here.

Reply:
Thanks for the feedback, @beenje. I removed it.